src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -1776,7 +1776,9 @@ internal SyntaxNode InterpolationFormatClause(string format)
     /// This is typically a null value for reference types or a zero-filled value for value types.
     /// </summary>
     public abstract SyntaxNode DefaultExpression(SyntaxNode type);
-    public abstract SyntaxNode DefaultExpression(ITypeSymbol type);
```

Comment:
note: it is safe to make an abstract method a non-virtual method in a shipped API.  That's because:

1. callers to this already use callvirt, which works fine on an instance method.
2. this is a type that is not publicly extendible (it has internal constructors).  So no one but us could have been overriding this.